src/integrationTest/java/uk/gov/hmcts/reform/hmc/BaseTest.java

```diff
@@ -57,6 +59,8 @@ public class BaseTest {
     protected SecurityUtils securityUtils;
     @Inject
     protected ApplicationParams applicationParams;
+    @MockBean
+    protected MessageSenderToTopicConfiguration messageSenderToTopicConfiguration;
```

Comment:
The reason tests are taking a ridiculous amount of time at the moment is because we're trying to send messages to a topic on a real service bus, but the service bus doesn't exist so it keeps retrying. We don't and can't verify anything more with the sending anyway while there's no service bus alternative for local - also supported by the fact everything except the next class in this PR still works with this change.